pyteal/__init__.pyi

```diff
@@ -138,6 +138,7 @@ __all__ = [
     "Or",
     "Pop",
     "Reject",
+    "Replace",
```

Comment:
@jdtzmn Unrelated to line:  While reviewing the PR, I added a couple of tests in https://github.com/jdtzmn/pyteal/pull/1.

Can you see if you feel it's worth pulling into the PR?  Otherwise, we can close the child PR.